syft/workers/base.py

```diff
@@ -112,7 +112,14 @@ def __init__(
         self.id = id
         self.is_client_worker = is_client_worker
         self.log_msgs = log_msgs
-        self.verbose = verbose
+        if verbose is None:
+            self.verbose = hook.verbose if hasattr(hook, "verbose") else False
+        else:
+            self.verbose = verbose
+
+        if isinstance(hook, sy.TorchHook) and hasattr(hook, "_virtual_workers"):
```

Comment:
It seems like `hasattr(hook, "_virtual_workers")` is always `True`, given that the hook sets `self._virtual_workers = {self.local_worker}`?

Reply:
That would make it fail the first time while instantiating the "me" / local_worker from the torch hook.
Like [here](https://github.com/Syzygianinfern0/PySyft/blob/e9ecd4594129f0f307ff7fefaae136cbab8769ad/syft/frameworks/torch/hook/hook.py#L134)